src/test/java/com/wikia/webdriver/common/dataprovider/TrackingOptInDataProvider.java

```diff
@@ -6,16 +6,15 @@
 
 public class TrackingOptInDataProvider {
 
-  private static final String
-      ADS_KIKIMORA_PATTERN =
-      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*";
+  private static final String ADS_KIKIMORA_OPT_IN_ACCEPTED_PATTERN =
```

Comment:
FYI: the `u` parameter may include a `user_id` if a user is logged-in and accepts displaying ads (or visits the main page).